README.md

```diff
@@ -46,8 +46,11 @@ commit [0360811][].  It was created via `git clone --bare` and `git push
 [0360811]: https://github.com/sstephenson/bats/commit/03608115df2071fff4eaaff1605768c275e5f81f
 
 ## Table of contents
+
 - [Installation](#installation)
   - [Supported Bash versions](#supported-bash-versions)
+  - [macOS](#macos)
```

Comment:
Mind titling this as brew or homebrew? The other installation methods are titled via the "how", not the host platform. (Seems weird that "macOS" and "npm" would be alternative installation methods since installing via npm is one possible installation method _for_ macOS. As would be installing from source.)

Reply:
Good point; done. Will merge once the builds finish passing.